flask_rebar/rebar.py

```diff
@@ -132,12 +134,13 @@ def wrapped(*args, **kwargs):
             return rv
 
         data, status_code, headers = _unpack_view_func_return_value(rv)
+        headers.update(response_headers or {})
 
         schema = marshal_schema[status_code]  # May raise KeyError.
 
         # The schema may be declared as None to bypass marshaling (e.g. for 204 responses).
         if schema is None:
-            return make_response((data or "", status_code, headers))
+            return response(data=data or {}, status_code=status_code, headers=headers)
```

Comment:
I see that we later replace `{}` with an empty string... but I'm not sure why?

An empty object is not empty content. I.e. if I have a marshal schema of {204: None}, this looks like it adds '{}' to the response content. Which wouldn't be right.

Reply:
I replaced with an empty string to match the existing unit test:
https://github.com/plangrid/flask-rebar/blob/master/tests/test_rebar.py#L319